distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java

```diff
@@ -135,8 +135,7 @@ public void startManagedService(String servicePid, Map<String, Object> propertie
     @Override
     public void stopManagedService(String servicePid) throws IOException {
         Configuration sourceConfig = adminConfig.getConfiguration(servicePid, null);
-        ServiceManagerImpl.ServiceConfigurationListener listener =
-                new ServiceManagerImpl.ServiceConfigurationListener(sourceConfig.getPid());
+        new ServiceManagerImpl.ServiceConfigurationListener(sourceConfig.getPid());
```

Comment:
❗️ This looks like a bug actually. Pretty sure we should be removing the listener since we add it in `startManagedService()`.